index.html

```diff
@@ -0,0 +1,152 @@
+<!DOCTYPE html>
+<html>
+    <head>
+<meta charset="UTF-8">
+<meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Portfolio project</title>    
+  <link rel="stylesheet" href="styles.css">
+  <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'>
+</head>
+    <body>
+        <header>
+            <nav>
+                <ul>
+              <li><a class="logolink" href="">My Logo</a></li>
+              <li class="leftli"><a href=""><img class="imageone" src="hammenu.png" alt="ham"></a></li>
+                </ul>
+            </nav>
+        </header>
+        <div class="headline">
+            <div class="textdiv"><h1>I’m Alexandra <br>
+                Glad to see you!</h1>
+                <p>I’m a software developer! I can help you build a product , feature or website Look through 
+                  some of my work and experience! If you like what you see and have a project you need coded,
+                   don’t hestiate to contact me.</p>
+                   <h3 class="pcol">LET’S CONNECT</h3>
+                    <ul class="iconlist">
+                        <li><img class="icons" src="twitter.svg" alt="twitter"></li>
+                        <li><img class="icons" src="linkedinicon.svg" alt="linkedin"></li>
+                        <li><img class="icons" src="m.svg" alt="mimage"></li>
+                        <li><img class="icons" src="github.svg" alt="github"></li>
+                        <li><img class="icons" src="victory.svg" alt="victory"></li>
+                    </ul>
+                </div>
+            </div>
+        <div class="container">
+            <div class="imgdiv">
+                <img class="projects" src="image1.png" alt="Nature">
+            </div>
+            <div class="tonic">
+                    <h1 class="popheading">Tonic</h1>
+                    <p class="popp13">CANOPY <img class="dot" src="dot.png" alt="dot"> 
+                        <span class="popp13 col">Back End Dev</span> <img class="dot" src="dot.png" alt="dot"> 
+                         <span class="popp13 col">2015</span></p>
+                    <br>
+                    <p class="popp15">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <span class="htcj">HTML</span> 
+                    <span class="htcj">CSS</span>
+                    <span class="htcj">Javascript</span><br><br>
+                      <div class="btn">
+                        <a href="image1.png" target="_blank">See Project</a>
+                      </div>
+            </div>    
+           </div>
+           <div class="container2">
```

Comment:
- [ ] [OPTIONAL] It would be nice and semantic if you change the `div` tag to a `section` tag because the container of the section should be a `section` tag 👍🏼